tests/functional/test_show.py

```diff
@@ -151,6 +152,7 @@ def test_pip_show_is_short(script):
     assert len(lines) <= 10
 
 
+@pytest.mark.network
```

Comment:
Ok, I'll submit follow-up later then. 

Appveyor build has failed for absolutely unrelated reason, can I somehow restart only Appveyor, without Travis? I'd like not to wait 40 minutes for the new Travis build.